docs/developer/transform.md

```diff
@@ -15,6 +15,16 @@ This page shows how to:
 
 After that, you might want to make your code transform into an [Autofix](autofix.md).
 
+There's also an important capability called "dry run transform":
+Performing a transform on a branch, and then either raising either a PR or
+an issue, depending on build success or failure. This allows us to
+safely apply transforms across many repositories. There's a simple wrapper
+function to enable this.
+
+!!! tip
+    Dry run transforms are another example of how commands and events can work
+    hand in hand with Atomist to provide a uniquely powerful solution.
```

Comment:
can we add, "The pull request is created in response to the transform command. It is merged in response to a successful build event.

Reply:
👍